packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -697,7 +697,13 @@ IFluidDataStoreChannel, IFluidDataStoreRuntime, IFluidHandleContext {
                 // (i.e. it has a base mapping) - then we go ahead and summarize
                 return isAttached;
             }).map(async ([contextId, context]) => {
-                const contextSummary = await context.summarize(fullTree, trackState);
+                // If BlobAggregationStorage is engaged, we have to write full summary for data stores
```

Comment:
FYI - Opened https://github.com/microsoft/FluidFramework/issues/5165 to track this issue of incremental snapshots